starter.ps1

```
@@ -0,0 +1,893 @@
+                return $true
+            }
+        }
+        catch {
+            try {
+                $response = Invoke-WebRequest -Uri "http://127.0.0.1:8005/" -UseBasicParsing -TimeoutSec 2 -ErrorAction SilentlyContinue
+                if ($response.StatusCode -eq 200) {
+                    Write-Host "[OK] Backend service is ready!" -ForegroundColor Green
+                    return $true
+                }
+            }
+            catch { }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)